WalletWasabi/Crypto/Transcript.cs

```diff
@@ -0,0 +1,115 @@
+using NBitcoin.Secp256k1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using WalletWasabi.Crypto.Groups;
+using WalletWasabi.Crypto.Randomness;
+using WalletWasabi.Crypto.StrobeProtocol;
+
+namespace WalletWasabi.Crypto.ZeroKnowledge
```

Comment:
path is ` WalletWasabi/Crypto/Transcript.cs` but fully qualified class name is `WalletWasabi.Crypto.ZeroKnowledge.Transcript`... should it be in ZeroKnowledge subdir?

Reply:
I think so